host/crypto/openssl/cert.c

```diff
@@ -636,15 +637,33 @@ oe_result_t oe_cert_chain_read_pem(
 
     /* Must have pem_size-1 non-zero characters followed by zero-terminator */
```

Comment:
Should this have the same comments as crypto/cert.c 742-744?

Reply:
Sure. Will make them similar